tests/observability/tests/pod_disruption_budget.go

```diff
@@ -0,0 +1,142 @@
+		err = globalhelper.LaunchTests(tnfTestCaseName, qeTcFileName)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Verify test case status in Junit and Claim reports")
+		err = globalhelper.ValidateIfReportsAreValid(tnfTestCaseName, globalparameters.TestCasePassed)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	// 56636
+	It("One deployment, pod disruption budget maxUnavailable value meet requirements", func() {
+		qeTcFileName := globalhelper.ConvertSpecNameToFileName(CurrentSpecReport().FullText())
+
+		By("Create deployment")
+		dep := deployment.DefineDeployment(tsparams.TestDeploymentBaseName, tsparams.TestNamespace,
+			globalhelper.Configuration.General.TestImage, tsparams.TnfTargetPodLabels)
+
+		deployment.RedefineWithReplicaNumber(dep, 2)
+
+		err := globalhelper.CreateAndWaitUntilDeploymentIsReady(dep, tsparams.DeploymentDeployTimeoutMins)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Create pod disruption budget")
+		pdb := poddisruptionbudget.DefinePodDisruptionBudgetMaxUnAvailable(tsparams.TestPdbBaseName, tsparams.TestNamespace,
+			intstr.FromInt(1), tsparams.TnfTargetPodLabels)
+
+		err = globalhelper.CreatPodDisruptionBudget(pdb, tsparams.PdbDeployTimeoutMins)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Start TNF " + tnfTestCaseName + " test case")
+		err = globalhelper.LaunchTests(tnfTestCaseName, qeTcFileName)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Verify test case status in Junit and Claim reports")
+		err = globalhelper.ValidateIfReportsAreValid(tnfTestCaseName, globalparameters.TestCasePassed)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	// 56637
+	It("One statefulSet, pod disruption budget minAvailable value is zero [negative]", func() {
+		qeTcFileName := globalhelper.ConvertSpecNameToFileName(CurrentSpecReport().FullText())
+
+		By("Create statefulSet")
+		sf := statefulset.DefineStatefulSet(tsparams.TestStatefulSetBaseName, tsparams.TestNamespace,
+			globalhelper.Configuration.General.TestImage, tsparams.TnfTargetPodLabels)
+
+		statefulset.RedefineWithReplicaNumber(sf, 1)
+
+		err := globalhelper.CreateAndWaitUntilStatefulSetIsReady(sf, tsparams.StatefulSetDeployTimeoutMins)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Create pod disruption budget")
+		pdb := poddisruptionbudget.DefinePodDisruptionBudgetMinAvailable(tsparams.TestPdbBaseName, tsparams.TestNamespace,
+			intstr.FromInt(0), tsparams.TnfTargetPodLabels)
+
+		err = globalhelper.CreatPodDisruptionBudget(pdb, tsparams.PdbDeployTimeoutMins)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Start TNF " + tnfTestCaseName + " test case")
+		err = globalhelper.LaunchTests(tnfTestCaseName, qeTcFileName)
+		Expect(err).To(HaveOccurred())
+
+		By("Verify test case status in Junit and Claim reports")
+		err = globalhelper.ValidateIfReportsAreValid(tnfTestCaseName, globalparameters.TestCaseFailed)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	// 56638
+	It("One deployment, pod disruption budget maxUnavailable equals to replica number [negative]", func() {
```

Comment:
Should we add a TC where the PDB max unavailable is > maxReplicas

Reply:
Added, thanks